hack/e2e/run-rp-and-e2e.sh

```diff
@@ -105,6 +105,7 @@ run_e2e() {
     az aro create \
       -g "$ARO_RESOURCEGROUP" \
       -n "$CLUSTER" \
+      --location ${LOCATION^} \
```

Comment:
comment? why are you upper casing the first character?

Reply:
I think this is done tin order o create a test case with wrong input and validate that the code handles it properly. I agree that a comment will be helpful. 

Would it be better if we stick to the same parameters format?  We use -n instead --name and -g instead of --resource-group, I think `-l` would be more appropriate here. or use the longer options within the same command

Reply:
Yes, this is to create a test case. Same as in https://github.com/Azure/ARO-RP/pull/587